docs/docs/app-builder/query-panel.md

```diff
@@ -3,153 +3,192 @@ id: query-panel
 
-#### Save
+### Query Parameters
 
-Save is used to save the changes whenever a change is made in query.
+Query Parameters are essential values that must be provided in a query for the server to generate a response. These parameters encompass **endpoints**, **methods**, or **operations**. It's important to note that the specific set of Query Parameters varies for each datasource.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/save.png" alt="App Builder: Component library- right sidebar"/>
+<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/newui2/params.png" alt="App Builder: Query Panel"/>
 
 </div>
 
-#### Run
+#### Datasource
 
-Run is used to trigger the query, running the query will interact with the application unlike `Preview`.
+The primary and default parameter found in all queries is **Datasource**. This option allows you to choose the appropriate datasource for your query.
+
+In cases where multiple datasources of the same type are connected, you can easily switch the query's datasource using the dropdown menu.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/run.png" alt="App Builder: Component library- right sidebar"/>
+<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/newui2/switch.png" alt="App Builder: Query Panel"/>
 
 </div>
 
-### Query Parameters
+### Transformation
 
-Query Parameters are the values required for the query to return a response from the server. Parameters include **endpoints**, **methods**, or **operations**. Query Parameters are different for each datasource.
+Transformations can be enabled on queries to transform the query results. ToolJet allows you to transform the query results using two programming languages JavaScript & Python. Check the detailed documentation on **[Transformations](/docs/tutorial/transformations)**.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/params.png" alt="App Builder: Component library- right sidebar"/>
+<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/newui2/transform.gif" alt="App Builder: Query Panel"/>
 
 </div>
 
-### Transformation
-
-Transformations can be enabled on queries to transform the query results. ToolJet allows you to transform the query results using two programming languages JavaScript & Python. Check the detailed documentation on **[Transformations](/docs/tutorial/transformations)**.
+### Settings
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/transform.png" alt="App Builder: Component library- right sidebar"/>
+<img className="screenshot-full" src="/img/v2-beta/app-builder/querypanel/newui2/settings.png" alt="App Builder: Query Panel"/>
 
 </div>
 
-### Advanced options
-
 #### Run this query on application load?
```

Comment:
I think this is duplicated. The 2nd one should be `Show notification on success?`
<img width="981" alt="image" src="https://github.com/ToolJet/ToolJet/assets/57667706/f5a6456c-d612-4fef-94e5-b05074923f7c">